### ISSUE TITLE
Inaccurate reading time for Chinese

### ISSUE THREAD

**Issue by Destaq** [Journaly/journaly#524]
All Chinese stories, regardless of length, have a `1 min. read` stamp on them, even if they are clearly above that time.

This makes it that much harder for those correcting to see the length of the text and doesn't accurately represent its true length.

Here you can see everything is as I said:
![image](https://user-images.githubusercontent.com/61620873/112329056-230eac00-8cb7-11eb-99ad-b0454873d1aa.png)

**Comment by Lanny** [comment]
Yup, related to #208 . Multi-lingual read times are hard, in part because word counts for languages like Chinese is hard. Will try to take a look at it relatively soon.
